packages/loader/container-loader/src/container.ts

```diff
@@ -2378,7 +2378,10 @@ export class Container
 		this.emit("op", message);
 	}
 
-	// unknown should be removed once `@alpha` tag is removed from IContainerContext
+	/**
+	 * `unknown` should be removed once `IContainerContext` is `@internal` or `@system`.
```

Comment:
Nit (to conform to the expectation that the summary portion of a TSDoc comment is the semantic description of the thing being documented):
```suggestion
	 * @privateRemarks `unknown` should be removed once `IContainerContext` is `@internal` or `@system`.
```